### PR TITLE
Add Location Control sources of information

### PR DESCRIPTION
**Pull request by sebastian-aranda**
This PR connects the `ControlLocation` model (https://github.com/lsst-ts/LOVE-manager/pull/179) to get the current set location of control. Also some refactor was done to the `ObservatorySummary` and `ObservatorySummaryMenu` components. Some missing telemetries were also added.

### CODE REVIEW

File: love/src/components/Layout/Layout.jsx [hunk 1]
```diff
@@ -294,18 +297,19 @@ class Layout extends Component {
...
-    switch (location) {
-      case 'Unknown':
+    switch (controlLocation ? controlLocation.name : 'unknown') {
+      case 'unknown':
         return <UnknownLocationIcon className={style} />;
-      case 'Tucson':
-        return <CactusIcon className={style} />;
-      case 'Base':
+      case 'tucson':
+        return <CactusIcon title="PRUEBA DE TITULO" className={style} />;
```
Comment (areyesd14) [inline love/src/components/Layout/Layout.jsx:306]: Is correct this title?

File: love/src/components/ObservatorySummary/ObservatorySummary.jsx [hunk 2]
```diff
@@ -1,56 +1,94 @@
...
           <Title wide>Simonyi Telescope</Title>
-          <Label>Summary</Label>
-          <Value>
-            <StatusText status={'invalid'}>UNKNOWN</StatusText>
-          </Value>
           <Label>Operation Mode</Label>
-          <Value>UNKNOWN</Value>
-          <Label>Obsv. Mode</Label>
-          <Value>{this.props.simonyiObservingMode}</Value>
+          <Value>Unknown</Value>
```
Comment (areyesd14) [inline love/src/components/ObservatorySummary/ObservatorySummary.jsx:63]: This "Unknown" value is correct?

File: love/src/components/ObservatorySummary/ObservatorySummary.jsx [hunk 3]
```diff
@@ -1,56 +1,94 @@
...
         <SummaryPanel>
           <Title wide>Vera C. Rubin Observatory</Title>
           <Label>Control</Label>
-          <Value>UNKNOWN</Value>
+          <Value>
+            <span title={`Last updated: ${lastUpdated?.toUTCString()}`}>{controlLocationName}</span>
+          </Value>
           <Label>Power Source</Label>
-          <Value>UNKNOWN</Value>
+          <Value>Unknown</Value>
```
Comment (areyesd14) [inline love/src/components/ObservatorySummary/ObservatorySummary.jsx:79]: This "Unknown" value is correct?

**Review by areyesd14 — APPROVED** [review]
Looks good! Thanks @sebastian-aranda !!

I left a few single comments about some "Unknown" values, but I think this is because we haven't yet this information.

File: love/src/components/Layout/Layout.jsx (hunk 1 above)
Comment (sebastian-aranda) [reply love/src/components/Layout/Layout.jsx:306]: thanks for noticing this 😅️. This is incorrect, I will remove it.

File: love/src/components/ObservatorySummary/ObservatorySummary.jsx (hunk 2 above)
Comment (sebastian-aranda) [reply love/src/components/ObservatorySummary/ObservatorySummary.jsx:63]: We still don't have the source of information for the Operation Mode, we are investigating where to get it. For the moment I think is ok to keep it like "Unknown"

File: love/src/components/ObservatorySummary/ObservatorySummary.jsx (hunk 3 above)
Comment (sebastian-aranda) [reply love/src/components/ObservatorySummary/ObservatorySummary.jsx:79]: Same as the previous comment, we still don't have this source of information.

**Pull request merged**